### PR TITLE
code refactoring

### PR DESCRIPTION
**Pull request by prayans**
1. Remove the API key from the comments in the google.ts
2. Export google utils too from the lib/index.tsx
3. Attach field key name should not be CamelCase, change it to the kebab-case
4. Expose everything in the lib/index.tsx which is exported in the whole package.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,23 +20,21 @@
   },
   "devDependencies": {
     "@date-io/core": "^2.10.6",
```
Comment (sirajalam049) [inline package.json]: What is the need for date-io/core here in this package?

File: src/lib/google.ts [hunk 2]
```diff
@@ -1,13 +1,3 @@
-/**
```
Comment (sirajalam049) [inline src/lib/google.ts]: Only the API-KEY was supposed to be removed. Comments should be there for any dev, and also add it to the ReadMe.md for users.

**Review by sirajalam049 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (prayans) [reply package.json]: @sirajalam049 resolve

File: src/lib/google.ts (hunk 2 above)
Comment (prayans) [reply src/lib/google.ts]: @sirajalam049 resolve

**Review by sirajalam049 — APPROVED** [review]

**Pull request merged**